Jenkinsfile

```diff
@@ -39,9 +39,33 @@ dockerTemplate {
         }
       }
 
+      stage('Test') {
+
+        // Spawn the image just created
+        mavenNode(mavenImage:image) {
+          container('maven') {
+            echo 'Container spawned successfully'
+            sh 'cat /etc/redhat-release'
+            sh 'hostname'
+            sh 'java -version'
+            sh 'mvn -version'
+            sh 'git --version'
+
+            // I've no idea where the canonical source for booster apps are, so
+            // let's clone a fork instead
+            sh 'git clone https://github.com/jaseemabid/vertxapp'
```

Comment:
can this be a config? also have you considered --shallow ?

Reply:
We can just change the url when we find out where to clone from. --shallow wont make a difference on this one because its just one or 2 commits there.

Reply:
Is this what you mean - https://github.com/openshiftio-vertx-boosters/vertx-http-booster ? please consider using a const instead of a hard coded value.

Reply:
@sthaha They are similar but quite different; see Jenkinsfile for example. I'm updating this anyway since I'm not using the changed files